Oqtane.Client/Modules/Admin/Profiles/Edit.razor

```diff
@@ -154,8 +154,14 @@
             profile.DefaultValue = _defaultvalue;
             profile.IsRequired = (_isrequired == null ? false : Boolean.Parse(_isrequired));
             profile.IsPrivate = (_isprivate == null ? false : Boolean.Parse(_isprivate));
-            profile = await ProfileService.UpdateProfileAsync(profile);
-            
+            if (_profileid != -1)
+            {
+                profile = await ProfileService.UpdateProfileAsync(profile);
+            }else
```

Comment:
Format

Reply:
we can use

```csharp
profile = _profileid != -1
    ? await ProfileService.UpdateProfileAsync(profile)
    : await ProfileService.AddProfileAsync(profile);
```

Reply:
My preference is to use the traditional conditional logic as it is much easier for developers to read and understand - which makes the code easier to maintain

Reply:
the else should be on its line to be consistent with the rest of the framework code, but I will merge this PR anyways.

Reply:
To be sure I comply correctly in the future, the desire is

`}

else

{`

Reply:
Correct ( without the extra line breaks before and after the “else” of course ). If you look at the other framework code you will are the standard is:

If ()
{
...
}
else
{
....
}